### DISCUSSION TITLE
请问如何修改vc-drawing组件的hover效果

### DISCUSSION THREAD

**Discussion by BerryLe** [zouyaoji/vue-cesium#436] [Q&A]
请问如何修改vc-drawing组件的hover效果

**Comment by zouyaoji** [comment]
是指鼠标移动到绘制的对象上的hover效果吗？比如移动到绘制的点上。

**Reply by BerryLe** [reply]
是要点击展开或收拢的 鼠标悬浮。悬浮提示展开或收拢。这个提示能修改吗？

**Reply by zouyaoji** [reply]
在mainFabOpts中修改哈。
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/components/drawings/src/defaultProps.ts#L106


https://github.com/zouyaoji/vue-cesium/blob/23bdad40d899dbbf6f255d1516b8d5f664c75d34/packages/composables/use-drawing/use-drawing-fab.ts#L444

**Reply by BerryLe** [reply]
改好了，谢谢。

**Reply by kankantian** [reply]
请问展开/收拢这个提示怎么关闭呢？

**Reply by zouyaoji** [reply]
关闭不了，可以改。

**Reply by sogeter** [reply]
> 是指鼠标移动到绘制的对象上的hover效果吗？比如移动到绘制的点上。

这个怎么实现
